src/analysis/normed_space/linear_isometry.lean

```diff
@@ -397,6 +397,13 @@ include σ₁₃ σ₂₁
 @[simp] lemma coe_trans (e₁ : E ≃ₛₗᵢ[σ₁₂] E₂) (e₂ : E₂ ≃ₛₗᵢ[σ₂₃] E₃) : ⇑(e₁.trans e₂) = e₂ ∘ e₁ :=
 rfl
 
+@[simp] lemma trans_apply (e₁ : E ≃ₛₗᵢ[σ₁₂] E₂) (e₂ : E₂ ≃ₛₗᵢ[σ₂₃] E₃) (c : E) :
+  (e₁.trans e₂ : E ≃ₛₗᵢ[σ₁₃] E₃) c = e₂ (e₁ c) :=
+rfl
+
+@[simp] lemma symm_trans_apply (e₁ : E ≃ₛₗᵢ[σ₁₂] E₂) (e₂ : E₂ ≃ₛₗᵢ[σ₂₃] E₃) (c : E₃) :
+  (e₁.trans e₂ : E ≃ₛₗᵢ[σ₁₃] E₃).symm c = e₁.symm (e₂.symm c) := rfl
```

Comment:
Normally we don't have this lemma, and instead have
```suggestion
@[simp] lemma symm_trans (e₁ : E ≃ₛₗᵢ[σ₁₂] E₂) (e₂ : E₂ ≃ₛₗᵢ[σ₂₃] E₃) :
  (e₁.trans e₂ : E ≃ₛₗᵢ[σ₁₃] E₃).symm = e₂.symm.trans e₁.symm := rfl
```
(untested).

Can you change this accordingly, so as to be consistent with all the other lemmas like this in mathlib?

Reply:
Actually nevermind, it seems we're not as consistent as I thought we were. Can you add both versions (keeping your version as `simp`)?

Reply:
In fact, we already have the other version...